### PR TITLE
Implement update inventory functionality, and write test cases.

### PR DESCRIPTION
**Pull request by mmclslwsl**
All lines added are covered in test cases.

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from sqlalchemy.exc import IntegrityError
 
 # Import utilities
-from service.utilities import check_content_type
+from service.utilities import check_content_type, check_condition_type
```
Comment (vkrishna1997) [inline service/routes.py:13]: Duplicate import of check_content_type

**Review by vkrishna1997 — CHANGES_REQUESTED** [review]
Can you rebase off Master. That should address my 1 comment. Otherwise looks good!
Edit: I'm dumb and misread your import. You are all good! Just rebase required.

File: service/models.py [hunk 2]
```diff
@@ -74,6 +74,12 @@ def update(self):
             raise DataValidationError("Update called with empty ID field")
         if not self.condition:
             raise DataValidationError("Update called with empty Condition field")
+                
+        # Update the attributes
+        self.quantity = self.quantity
```
Comment (vkrishna1997) [inline service/models.py:79]: Seems redundant? self = self?

**Review by vkrishna1997 — CHANGES_REQUESTED** [review]

**Review by IrregularHunterX — COMMENTED** [review]
The other changes look good to me

**Review by MarvinTellis — APPROVED** [review]

**Review by IrregularHunterX — APPROVED** [review]

**Pull request merged**